recipe/meta.yaml

```
@@ -230,119 +230,121 @@ outputs:
       license: Apache-2.0
       license_family: APACHE
       license_file: LICENSE.md
-      summary: The OmniSci database
+      summary: The HeavyDB database
 
       description: |
-        OmniSciDB is an in-memory, column store, SQL relational database
+        HeavyDB is an in-memory, column store, SQL relational database
         that was designed from the ground up to run on GPUs.
 
-        This recipe provides both CUDA enabled and CUDA disabled omniscidb
+        This recipe provides both CUDA enabled and CUDA disabled heavydb
         packages.
-      doc_url: https://www.omnisci.com/docs/latest/
-      dev_url: https://github.com/omnisci/omniscidb
+      doc_url: https://docs.heavy.ai/
+      dev_url: https://github.com/heavyai/heavydb
 
-  - name: pyomniscidb
+  - name: pyheavydb
     version: {{ version }}
-    script: build-pyomniscidb.sh   # [linux64]
-    script: build-pyomniscidb.bat  # [win]
     build:
       skip: true  # [cuda_compiler_version in ("11.0",)]
-      string: {{ py_build_string }}
       noarch: python
+      script: python -m pip install -vv --no-deps python/.
       run_exports:
-        - {{ pin_subpackage('pyomniscidb',  max_pin='x.x.x') }}
+        - {{ pin_subpackage('pyheavydb',  max_pin='x.x.x') }}
     requirements:
       host:
-        - python
-        - setuptools_scm
-        - packaging >=20.0
+        - pip
+        - python >=3.7
+        - flit-core
       run:
-        - python
-        - setuptools
-        - {{ pin_compatible('numpy', lower_bound='1.16') }}
+        - python >=3.7
+        - importlib_metadata
+        - packaging
+        - numpy >=1.16
         - thrift {{ pythrift_version }}
-        - packaging >=20.0
         - sqlalchemy >=1.3
-        - requests
+        - requests >=2.23.0
 
     about:
-      home: http://github.com/omnisci/pyomnisci
+      home: https://github.com/heavyai/heavydb
       license: Apache-2.0
       license_family: Apache
       license_file: ./python/LICENSE.md
-      summary: A python DB API 2 compatible client for OmniSci (formerly MapD).
+      summary: A python DB API 2 compatible client for HeavyDB (formerly OmniSci and MapD).
       description: |
-        A python DB API 2 compatible client for OmniSci (formerly MapD).
-      doc_url: https://pyomnisci.readthedocs.io
-      dev_url: https://github.com/omnisci/pyomnisci
+        A python DB API 2 compatible client HeavyDB (formerly OmniSci and MapD).
+      doc_url: https://pyheavydb.readthedocs.io
+      dev_url: https://github.com/heavyai/heavydb
 
     test:
       imports:
-        - omnisci.cursor
+        - heavydb
+      commands:
+        - pip check
+      requires:
+        - pip
 
-  - name: omniscidbe
+  - name: heavydbe
     version: {{ version }}
-    script: build-omniscidb.sh
+    script: build-heavydb.sh
     build:
       string: {{ build_string_ext }}
-      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0") or not linux64]
+      skip: true
```

Same here, keep the original skip:
```suggestion
      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0") or not linux64]
      skip: true
```